template.yaml

```diff
@@ -924,19 +837,120 @@ Resources:
   # Start: Audit delivery stream resources #
   ##########################################
 
+  # Lambda function resources
+  DelimiterLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      KmsKeyId: '{{resolve:ssm:LogsKmsKeyArn}}'
+      LogGroupName: /aws/lambda/DelimiterFunction
+      RetentionInDays: 7
+
+  S3DelimiterLogs:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      KmsKeyId: '{{resolve:ssm:LogsKmsKeyArn}}'
+      LogGroupName: !Sub /aws/lambda/${AWS::StackName}-s3-delimiter
```

Comment:
i've never liked the name `delimiter`. what its doing is flattening the json object to make https://jsonlines.org/examples/.

if we look at the architecture design, its an event consumer processor so maybe call the function `AuditConsumerProcessor` or `S3ConsumerProcessor`

Reply:
Renamed to 'audit-message-delimiter` after some discussion